### PR TITLE
New API for adding attachment to test run (test cycle)

### PR DESCRIPTION
**Pull request by shrikantsingh0585**

### CODE REVIEW

**Comment by 2Fake** [comment]
Looks good to me. I'm wondering why the workflows are not run

**Review by 2Fake — APPROVED** [review]

**Comment by 2Fake** [comment]
@Shutgun Could you please have a look too as this is mostly your project

File: adaptavist/adaptavist.py [hunk 1]
```diff
@@ -887,6 +887,24 @@ def add_test_result_attachment(
             else self._upload_file(request_url, attachment, filename)
         )
 
+    def add_test_run_attachment(
+        self, test_run_key: str, attachment: Union[str, BinaryIO], filename: str = ""
+    ) -> bool:
```
Comment (2Fake) [inline adaptavist/adaptavist.py:892]: I ran the workflows on my local machine. Please change this line to this.

```suggestion
        self, test_run_key: str, attachment: Union[str, BinaryIO], filename: str = "" ) -> bool:
```

**Comment by 2Fake** [comment]
A unittest is missing. Could you please add one? It should be enough if you test, that the correct function is called.

**Comment by Shutgun** [comment]
@2Fake , I will but not before Saturday.

**Review by Shutgun — APPROVED** [review]
I guess, this is good to go. @2Fake ?

**Review by 2Fake — APPROVED** [review]

**Pull request merged**